src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -1422,15 +1422,19 @@ public void SymmetricExceptWith(IEnumerable<T> other)
             }
             else
             {
-                T[] elements = EnumerableHelpers.ToArray(other);
-                Array.Sort(elements, this.Comparer);
-                SymmetricExceptWithSameEC(elements);
+                int length;
+                T[] elements = EnumerableHelpers.ToArray(other, out length);
+                Array.Sort(elements, 0, length, Comparer);
+                SymmetricExceptWithSameEC(elements, length);
             }
         }
 
         //OTHER must be a set
```

Comment:
Should this comment be removed now that there's an assertion to the same effect? To my mind that assertion tells someone reading the code the same thing as this, and does so better.